### PR TITLE
 Have autotransfer accept turrets

### PR DESCRIPTION
**Pull request by sbxte**

### CODE REVIEW

File: core/src/mindustry/client/utils/AutoTransfer.kt [hunk 1]
```diff
@@ -1,12 +1,15 @@
 package mindustry.client.utils
 
-import arc.*
```
Comment (buthed010203) [inline core/src/mindustry/client/utils/AutoTransfer.kt]: Why the imports bad

Comment (sbxte) [reply core/src/mindustry/client/utils/AutoTransfer.kt]: Intellij import optimizer

Comment (sbxte) [reply core/src/mindustry/client/utils/AutoTransfer.kt]: Guess I'll revert it 🤷

**Comment by buthed010203** [comment]
The problem with turrets is that they dont sync their item counts so you can end up in a situation where you try to load a turret on loop despite the turret being full already on the server, its pretty annoying

**Comment by sbxte** [comment]
hmm

**Comment by sbxte** [comment]
Yeah but I don't think that'd be very noticeable. Since there's the ratelimiter, players can resync, and are unlikely to always stay at the same core and nearby the same turrets all the time. Even then the turrets can be deconstructed/reconstructed which would resync.

**Comment by buthed010203** [comment]
There was a bug preventing this from working, its now been fixed

**Pull request closed without merging**